lib/amazon_business_api/invoice_detail/deserializer.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative '../invoice_issuer_detail/deserializer'
+require_relative '../physical_address/deserializer'
+require_relative '../merchant_detail/deserializer'
+require_relative '../tax_detail/deserializer'
+require_relative '../transaction_data_item/deserializer'
+require_relative '../invoice_level_fee_detail/deserializer'
+
+module AmazonBusinessApi
+  class InvoiceDetail
+    class Deserializer < AmazonBusinessApi::Deserializer
+      attribute :invoice_number, hash_attribute: :invoiceNumber
+      attribute :invoice_date, hash_attribute: :invoiceDate
+      attribute :invoice_type, hash_attribute: :invoiceType
+      attribute :associated_invoice_number, hash_attribute: :associatedInvoiceNumber
+      attribute :associated_invoice_date, hash_attribute: :associatedInvoiceDate
+      references_one :invoice_issuer, deserializer: InvoiceIssuerDetail::Deserializer, hash_attribute: :invoiceIssuer
+      references_one :bill_to_address, deserializer: PhysicalAddress::Deserializer, hash_attribute: :billToAddress
+      references_one :seller_information, deserializer: MerchantDetail::Deserializer, hash_attribute: :sellerInformation
+      attribute :order_date, hash_attribute: :orderDate
+      attribute :currency_code, hash_attribute: :currencyCode
+      attribute :invoice_total, hash_attribute: :invoiceTotal
+      references_one :tax, deserializer: TaxDetail::Deserializer, hash_attribute: :tax
+      # references_many :transaction_data_items, deserializer: TransactionDataItem::Deserializer, hash_attribute: :transactionDataItems
+      # references_many :fees, deserializer: InvoiceLevelFeeDetail::Deserializer, hash_attribute: :fees
+    end
+  end
+end
+
+# CF vim /home/michou/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/gems/ledger_sync-2.6.0/lib/ledger_sync/serialization/deserializer_attribute.rb
+# CF vim /home/michou/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/gems/ledger_sync-2.6.0/lib/ledger_sync/error/resource_errors.rb
```

Comment:
Here, for references_many, just for these two (`transaction_data_items` and `fees`), it raises an error.

Yeah, these references are not present in the response payload, but the other ones neither.

It should ignore it like it always did. But on these two, it raises an error. I don't understand why.

Did I do a typo? Can you check it? I don't have any answer. I used a lot of debug, even in gems files.

90% of the resources in this file are never received. Docs said they are only for JP area, not UK or USA.